cmd/launcher/interactive.go

```diff
@@ -35,11 +37,17 @@ func runInteractive(args []string) error {
 
 	osquerydPath := *flOsquerydPath
 	if osquerydPath == "" {
-		osquerydPath = launcher.FindOsquery()
-		if osquerydPath == "" {
-			return errors.New("Could not find osqueryd binary")
+		latestOsquerydBinary, err := tuf.CheckOutLatestWithoutConfig("osqueryd", log.NewNopLogger())
+		if err != nil {
+			osquerydPath = launcher.FindOsquery()
```

Comment:
AFAIK `interactive` isn't heavily used outside us, is it any cleaner to remove this fallback behavior?